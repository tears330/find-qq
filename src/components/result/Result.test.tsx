import { render } from '@testing-library/react';
import { Result } from '.';
describe('Result', () => {
  it('render result', () => {
    const { baseElement, rerender } = render(<Result msg="test" />);
    expect(baseElement).toMatchSnapshot();

    rerender(<Result msg="test2" />);

    expect(baseElement).toMatchSnapshot();
  });
});
