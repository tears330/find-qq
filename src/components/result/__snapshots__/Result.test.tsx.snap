// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result render result 1`] = `
<body>
  <div>
    <div
      class="com-result"
    >
      <p>
        Woops! Something went wrong :(
      </p>
      <p>
        test
      </p>
    </div>
  </div>
</body>
`;

exports[`Result render result 2`] = `
<body>
  <div>
    <div
      class="com-result"
    >
      <p>
        Woops! Something went wrong :(
      </p>
      <p>
        test2
      </p>
    </div>
  </div>
</body>
`;
